parm/config/gfs/config.base

```diff
@@ -478,4 +478,11 @@ export OFFSET_START_HOUR=0
 # Number of regional collectives to create soundings for
 export NUM_SND_COLLECTIVES=${NUM_SND_COLLECTIVES:-9}
 
+# The tracker, genesis, and METplus jobs are not supported on AWS yet
+if [[ "${machine}" == "AWSPW" ]]; then
+   export DO_TRACKER="NO"
+   export DO_GENESIS="NO"
+   export DO_METP="NO"
+fi
```

Comment:
This is fine for now, but we should add a `TODO` here to place these in `workflow/hosts/awspw.yaml`

Reply:
"TODO" added as suggested.